### PR TITLE
activate server span

### PR DESCRIPTION
**Pull request by malafeev**
fix for #31

### CODE REVIEW

File: src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java [hunk 1]
```diff
@@ -124,7 +124,8 @@ public ServerServiceDefinition intercept(BindableService bindableService) {
 
     final String operationName = operationNameConstructor
         .constructOperationName(call.getMethodDescriptor());
-    final Span span = getSpanFromHeaders(headerMap, operationName);
+    final Scope scope = getSpanFromHeaders(headerMap, operationName);
+    final Span span = scope.span();
```
Comment (yurishkuro) [inline src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java]: scope.span() is being deprecated. I would recommend keeping getSpanFromHeaders() as is (i.e. returning Span), and then activating that span here via `tracer.scopeManager.activate(span, false)`, which is the new/recommended pattern (the `false` param is also deprecated in 0.32).

File: src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java [hunk 2]
```diff
@@ -174,43 +175,36 @@ public void onMessage(ReqT message) {
         if (streaming || verbose) {
           span.log(ImmutableMap.of("Message received", message));
         }
-        try (Scope ignored = tracer.scopeManager().activate(span, false)) {
-          delegate().onMessage(message);
-        }
```
Comment (yurishkuro) [inline src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java]: can this function execute on a different thread?  If so, the activation above was correct and should not be removed.

File: src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java (hunk 1 above)
Comment (malafeev) [reply src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java]: got it, fix in another PR

File: src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java (hunk 2 above)
Comment (malafeev) [reply src/main/java/io/opentracing/contrib/grpc/ServerTracingInterceptor.java]: you are right, fixed in next PR

**Pull request merged**